From ed6c39f59e7be167e3b42261f1d298b86f4bae4d Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Fri, 5 Apr 2019 15:19:09 +0300
Subject: [PATCH 23/40] ls1043avnprdb: remove the QSGMII PHY addresses

There are no QSGMII interfaces on the LS1043A VNP-RDB.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
---
 include/configs/ls1043avnprdb.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/include/configs/ls1043avnprdb.h b/include/configs/ls1043avnprdb.h
index 6db55bb916..a3d818e6b9 100755
--- a/include/configs/ls1043avnprdb.h
+++ b/include/configs/ls1043avnprdb.h
@@ -289,11 +289,6 @@
 #define RGMII_PHY1_ADDR			0x1
 #define RGMII_PHY2_ADDR			0x2
 
-#define QSGMII_PORT1_PHY_ADDR		0x4
-#define QSGMII_PORT2_PHY_ADDR		0x5
-#define QSGMII_PORT3_PHY_ADDR		0x6
-#define QSGMII_PORT4_PHY_ADDR		0x7
-
 #define FM1_10GEC1_PHY_ADDR		0x1
 
 #define CONFIG_ETHPRIME			"FM1@DTSEC3"
-- 
2.17.1

